libkbfs/root_metadata.go

```diff
@@ -1089,6 +1089,9 @@ func (rmds *RootMetadataSigned) MakeFinalCopy(
 	}
 	// Set the final flag.
 	newBareMd.SetFinalBit()
+	// Set the copied bit, so that clients don't take the ops and byte
+	// counts in it seriously.
+	newBareMd.SetWriterMetadataCopiedBit()
```

Comment:
Would prefer a libkbfs test, since docker tests are pretty heavyweight, but :+1: on doing it separately later.